### PR TITLE
230 retrieve session on authentication

### PR DESCRIPTION
**Pull request by joshherkness**
Retrieve any current sessions for a user upon successful authentication.

### CODE REVIEW

File: vue/src/components/Authentication.vue [hunk 1]
```diff
@@ -134,6 +135,16 @@ export default {
         // Update the local user
         this.updateUser(user)
 
+        // Update the local session, if any exist for the current user
+        API.getCurrentSession().then((session) => {
+          if (session) {
+            this.setSession(session)
+          }
+        }).catch((err) => {
```
Comment (kdjohnson) [inline vue/src/components/Authentication.vue]: Have eslint ignore this line.

**Review by kdjohnson — CHANGES_REQUESTED** [review]

**Comment by joshherkness** [comment]
There we go.

**Review by kdjohnson — APPROVED** [review]

**Comment by kdjohnson** [comment]
@jjrobertson14 Have you tried this?

**Review by jjrobertson14 — APPROVED** [review]
The changes to tests.py and server.py look fine, and the tests for read_session_user_id still pass

**Pull request merged**